doc/source/serve/http-guide.md

```diff
@@ -1,12 +1,37 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you:
+
```

Comment:
This section helps you to understand how to:
- configure HTTP server location
- send HTTP request to serve deployment
- use Ray Serve integration with FastAPI
- use customized HTTP Adapter